server/boot/react.js

```diff
@@ -32,6 +33,10 @@ export default function reactSubRouter(app) {
 
   // These routes are in production
   routes.forEach((route) => {
+    if (route.startsWith('/settings')) {
+      router.get(route, ifNoUserRedirectTo('/map'), serveReactApp);
+      return;
+    }
```

Comment:
It was intentional to have the settings page always rendered regardless of whether the user is logged in or not. We will eventually move to a static site and we won't have the benefit of intercepting user requests for pages. This issue should be fixed client side. I think the fix in Settings.jsx is worth getting in, though.

Reply:
Hmmm, okay. I'll see if I have time to get together a client side fix for this in the next couple days -- if not I'll pull that part out and leave the Settings.jsx fix for merging.